### ISSUE TITLE
Dashboard - Add benchmark/metric description when filing bugs.

### ISSUE THREAD

**Issue by simonhatch** [catapult-project/catapult#3690]
@anniesullie @nedn @eakuefner @benshayden 

From Chrome Memory + Speed-Ops PRD doc
"Every benchmark and every metric should have a description that can be easily understood by any Chrome developer. This information should be linked when the crbug template."

These live here currently: https://chromium.googlesource.com/chromium/src/+/master/docs/speed/benchmark_harnesses/

I think ideally these would be sparsediagnostics that Dashboard/Pinpoint could use to grab this info, maybe in the short-term we can just store them in stored_object?

**Comment by anniesullie** [comment]
Do you think in the short term we could just extend the code that adds the memory documentation?

**Comment by simonhatch** [comment]
Yep, it's all done in bisect_report, it's a bit hardcoded but we could clean it up and just have a dict in there as an alternative.
